toolkit/themes/shared/design-system/build/css/tokens-platform.css

```diff
@@ -0,0 +1,4 @@
+:root {
```

Comment:
Same thing here, do we need to add  `@import url("chrome://global/skin/design-system/tokens-shared.css");` before the formatted variables?